frontend/src/components/NavBar/index.tsx

```diff
@@ -140,14 +177,25 @@ function NavBar() {
 
   const handlePanelClick = (panel: Panel) => {
     dispatch(setTabValue(panel));
+    if (panel === Panel.Dashboard) {
+      history.push('/dashboard');
+    } else if (location.pathname !== '/') {
+      history.push('/');
+    }
   };
```

Comment:
Can we think of a more future proof approach? What happens if we have another page? eg. /alerts or /batch-print ?